src/iidm/converter/Parameter.cpp

```diff
@@ -0,0 +1,56 @@
+/**
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+#include <powsybl/iidm/converter/Parameter.hpp>
+
+namespace powsybl {
+
+namespace iidm {
+
+namespace converter {
+
+Parameter::Parameter(const std::string& name, const ParameterType& type, const std::string& description, const std::string& defaultValue) :
+    m_type(type), m_description(description), m_defaultValue(defaultValue) {
```

Comment:
Coding style: one assignment per line. This remark applies for all the constructors